releasenotes/notes/tagger-kubernetes_pvc_tags-configuration-bd3746df3af77bf3.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Implement the kubernetes_persistent_volume_claims_as_tags configuration that allows
+    users to disable persistentvolumeclaim for Kubernetes resources.
```

Comment:
```suggestion
    users to disable PersistentVolumeClaim for Kubernetes resources.
```

Reply:
Hey @cswatt 👋 Thanks for the review! On this I think we should keep it as is as this is the "official" naming from Kubernetes: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims